corefreq-cli.c

```diff
@@ -600,8 +600,8 @@ TGrid *PrintRatioFreq(	Window *win, struct FLIP_FLOP *CFlop,
 			CUINT width, CELL_FUNC OutFunc, ATTRIBUTE attrib[])
 {
 	TGrid *pGrid = NULL;
-
-    if ((( (*pRatio) > 0)  && !zerobase) || (( (*pRatio) >= 0) && zerobase))
+	        /*EGA: *pRatio is unsigned (*pRatio) >= 0) is always true*/
```

Comment:
Indeed, most conditions can be fixed this way.
I will however measure the impacts of the **offset** variables which go up to the diver